src/lineup.test.js

```diff
@@ -0,0 +1,25 @@
+const test = require('ava');
+const sinon = require('sinon');
+const axios = require('axios');
+
+const lineup = require('./lineup');
+const tvheadendApi = require('./tvheadendApi');
+
+test('calls the API with the right options', async t => {
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
test('calls the API with the right options', async (t) => {
```